Sources/MongoSwift/ChangeStream.swift

```diff
@@ -0,0 +1,165 @@
+import mongoc
+
+/// A token used for manually resuming a change stream. Pass this to the `resumeAfter` or `startAfter` fields of
+/// `ChangeStreamOptions` to resume or start a change stream where a previous one left off.
+/// - SeeAlso: https://docs.mongodb.com/manual/changeStreams/#resume-a-change-stream
+public struct ChangeStreamToken: Codable {
+    private let resumeToken: Document
+
+    public init(resumeToken: Document) {
```

Comment:
does this need to be publicly initializable, given that users will be reading instances of this type from the `ChangeStream.resumeToken` property?

Reply:
So far, this is only used internally, so I will make it `internal`